scripts/node/node_template.yaml

```diff
@@ -99,7 +99,7 @@ spec:
           - /bin/sh
           - -c
           - |
-${SCRIPT}
+            ${SCRIPT}
```

Comment:
The scripts are all indented, so I don't think this is necessary. Would be great to remove the need for indentation in the scripts though.